src/schema/v2/artwork/index.ts

```diff
@@ -1986,6 +1991,12 @@ export const artworkResolver = async (_source, args, context, resolveInfo) => {
     resolveInfo
   )
 
+  const hasRequestedCollectorSignals = isFieldRequested(
+    "collectorSignals",
+    resolveInfo
+  )
```

Comment:
Followed nearby pattern here. I did not do the same for `filterArtworksConnection` yet - just marked it temp in the comments while naively always requesting collector signals.